index.html

```diff
@@ -18,81 +18,82 @@
       <div class="logo">
         <a class="logo-text" href="#">My Logo</a>
       </div>
+      <nav>
+        <ul class="desc-nav">
+          <li><a href="#">Portoflio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
Great job @fullstop125 🤝

- [ ] You may please address the nav section for the desktop version correctly in your media queries. Please take a look at the screenshots below.
- Your design
![Your navbar](https://user-images.githubusercontent.com/63917725/178038741-77b27de1-e4bd-4132-9ead-2ad7dca139a5.PNG)

- Figma design
![Figma navbar](https://user-images.githubusercontent.com/63917725/178038784-17bd7be3-832e-49f0-a34b-8180ff06b13e.PNG)

Reply:
solved